pkg/vadmin/start_db_vc.go

```diff
@@ -78,5 +78,10 @@ func (v *VClusterOps) genStartDBOptions(s *startdb.Parms, certs *HTTPSCerts) (vo
 	*opts.UserName = vapi.SuperUser
 	opts.Password = &v.Password
 	*opts.HonorUserInput = true
+
+	// timeout option
+	if v.VDB.Spec.RestartTimeout != 0 {
```

Comment:
Do we have different default timeouts depending if its admintools or vclusterops. If I am reading the code correctly the vclusterops default is 5 minutes but with admintools it is 20 minutes. Can we have them in sync? Otherwise it will be hard to document this parameter. My preference is to change this on the vcluster side rather than fix it in the operator.

Reply:
Yes, we should have them in sync, I update it in my Vcluster PR https://git.verticacorp.com/projects/VER/repos/server/pull-requests/19134/overview.